Soccar [UI]/src/nl/soccar/ui/fx/controller/CreateRoomFXMLController.java

```diff
@@ -154,14 +155,14 @@ private void createRoom() {
 
     private boolean checkInput(String roomName, String password) {

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused method parameter "password". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS1172)